OBAKit/UI/OBATheme.h

```diff
@@ -8,6 +8,11 @@
 
 @import UIKit;
 
+typedef enum {
+    OBAAppearanceNavBarTitleViewStyleDefault, // subtitle won't be displayed
+    OBAAppearanceNavBarTitleViewStyleSubtitle,
+} OBAAppearanceNavBarTitleViewStyle;
```

Comment:
Nice work on prefixing the enum values with the name of the enum! I know lots of developers with years of experience who still don't do this! 👏